docs/framework/20sources/30implementation/pause.md

```diff
@@ -1,14 +1,45 @@
 ### Ability to Pause Project
```

Comment:
Is this more of an operational concern than an implementation one? should this be moved? I can see arguments both ways, but it feels more like a source of operational risk than an implementation one to me.

Reply:
There is information about pausing (from a design perspective) in implementation, and then separately a section in operation, discussing the operational concerns.